tests/regression/gmail/test_firefox_gmail_ail_type_in_reply_field.sikuli/test_firefox_gmail_ail_type_in_reply_field.py

```diff
@@ -39,13 +39,13 @@
 app.wait_for_loaded()
 
 # Wait for stable
-sleep(5)
+sleep(2)
 
 # PRE ACTIONS
 app.click_first_mail()
```

Comment:
The `WebApp._click()` already wait for component exists.
So, in this case, it will
1. wait 'compose btn' <= `app.wait_for_loaded()`
2. wait 'more btn', and then click first mail <= `app.click_first_mail()`
3. wait 'reply btn', and then click reply btn <= `app.click_reply_btn()`

https://github.com/Mozilla-TWQA/Hasal/blob/dev/lib/sikuli/common.sikuli/common.py#L145